api/app/controllers/mno_enterprise/jpi/v1/admin/organizations_controller.rb

```diff
@@ -10,12 +10,16 @@ def index
         response.headers['X-Total-Count'] = @organizations.count
       else
         # Index mode
-        @organizations = MnoEnterprise::Organization
-        @organizations = @organizations.limit(params[:limit]) if params[:limit]
-        @organizations = @organizations.skip(params[:offset]) if params[:offset]
-        @organizations = @organizations.order_by(params[:order_by]) if params[:order_by]
-        @organizations = @organizations.where(params[:where]) if params[:where]
-        @organizations = @organizations.all.fetch
+        organizations = MnoEnterprise::Organization
+        organizations.limit(params[:limit]) if params[:limit]
+        organizations.skip(params[:offset]) if params[:offset]
+        organizations.order_by(params[:order_by]) if params[:order_by]
+        organizations.where(params[:where]) if params[:where]
+
```

Comment:
This is really confusing. I'm guessing this is some `Her` magic adding the filtering in place.

The previous code is a bit more understandable as it mirrors classic ActiveRecord